src/UI/StatusBar.re

```diff
@@ -23,6 +23,43 @@ module FontIcon = Oni_Components.FontIcon;
 module Diagnostics = Feature_LanguageSupport.Diagnostics;
 module Diagnostic = Feature_LanguageSupport.Diagnostic;
 module Editor = Feature_Editor.Editor;
+module Theme = Feature_Theme;
+
+module Colors = {
+  let transparent = Colors.transparentWhite;
```

Comment:
Ideally, instead of defining this here, this would be a key name corresponding to a contributed color. Unfortunately that's not feasible from the UI project. We'll need feature projects for that.